firedrake/slate/slac/compiler.py

```diff
@@ -213,7 +214,7 @@ def generate_kernel_ast(builder, statements, declared_temps):
                              qualifiers=["const"]))
 
     # Macro kernel
-    macro_kernel_name = "compile_slate"
+    macro_kernel_name = "pyop2_kernel_compile_slate"
```

Comment:
Is it still the case that all of these kernels need to be written with "pyop2_kernel_" for loopy to recognise them?